iothub_client/samples/iothub_client_sample_upload_to_blob/windows/iothub_client_sample_upload_to_blob.vcxproj

```diff
@@ -27,26 +27,26 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v140</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
```

Comment:
Why are we checking in Visual Studio project files? Shouldn't these be generated by CMake?

Reply:
This was added by JetstreamRoySprowl a few years ago. Good question is how this is being maintained.

Reply:
No. These have been there for a long time to make it easy for customers that want to run the samples using packages like nugets (now discontinued) and vcpkg.

Reply:
Do these reference the same source files that cmake uses or are they duplicated? 
We should think of moving these to Azure-Samples if they don't use the main build.

Reply:
These projects do not duplicate code perse, they use the code from the parent folder, which is the same "main.c" file used by cmake. There is no duplication.